concepticondata/conceptlists.tsv

```diff
@@ -382,4 +382,5 @@ Hantgan-2018-302	Hantgan, Abbie and List, Johann-Mattis	2018		302	basic	English
\ No newline at end of file
+Jonauskaite-2020-12	Jonauskaite, Domicele and Abu-Akel, Ahmad and Dael, Nele and Oberfeld, Daniel and Abdel-Khalek, Ahmed M. and Al-Rasheed, Abdulrahman S. and Antonietti, Jean-Philippe and Bogushevskaya, Victoria and Chamseddine, Amer and Chkonia, Eka and Corona, Violeta and Fonseca-Pedrero, Eduardo and Griber, Yulia A. and Grimshaw, Gina and Hasan, Aya Ahmed and Havelka, Jelena and Hirnstein, Marco and Karlsson, Bodil S. A. and Laurent, Eric and Lindeman, Marjaana and Marquardt, Lynn and Mefoh, Philip and Papadatou-Pastou, Marietta and Pérez-Albéniz, Alicia and Pouyan, Niloufar and Roinishvili, Maya and Romanyuk, Lyudmyla and Salgado Montejo, Alejandro and Schrag, Yann and Sultanova, Aygun and Uusküla, Mari and Vainio, Suvi and Wąsowicz, Grażyna and Zdravković, Sunčica and Zhang, Meng and Mohr, Christine	2020		12	specific, colors	English,Arabic,Azerbaijani,Chinese,Dutch,Estonian,Finnish,French,German,Georgian,Greek,Hebrew,Italian,Lithuanian,Norwegian,Persian,Polish,Russian,Serbian,Spanish,Swedish,Ukrainian	Global		Jonauskaite2020		The study investigates color-emotion associations by using the Geneva emotion wheel [(Scherer et al. 2013)](:bib:Scherer2013) in combination with 12 color terms across 22 languages.		
+Feleke-2021-150	Feleke, Tekabe Legesse	2021		150	specific, areal	English	Ethiosemitic languages	https://www.sciencedirect.com/science/article/pii/S2215039021000023	Feleke2021		The list was used in a study for the areal classification of Ethiosemitic languages and to establish distances/relationships between the members of this group. The items for the list were randomly selected. Note that the article mentions that the list contains 147 items, whereas the actual supplementary materials consist of 150 items.		
```

Comment:
the "randomly" selected sounds a bit strange, as this list seems ot be edited, so I'd add "according to the author"

Reply:
Done.